manifests/y/YggdrasilNetwork/Yggdrasil/0.5.4/YggdrasilNetwork.Yggdrasil.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: YggdrasilNetwork.Yggdrasil
+PackageVersion: 0.5.4
+PackageLocale: en-US
+Publisher: github.com/yggdrasil-network/yggdrasil-go

```

```suggestion
Publisher: github.com/yggdrasil-network
```
This should match control panel exactly